### PR TITLE
feat(convert) : add script allow to convert markdown to ascidoc

### PR DESCRIPTION
**Pull request by passga**
The script automize all manual step describes  needed to convert markdown to asciidoc ([see](https://github.com/bonitasoft/bonitasoft.github.io/blob/master/docs/migrate-from-old-documentation-site/doc-content-conversion-for-new-doc-site.adoc) ) 

the python script will :
 * sanitize markdown to render the markdown more convertible
 * convert markdown to acsiidoc.
 * sanitize the acsiidoc.
 * delete the markdown file

### CODE REVIEW

**Comment by benjaminParisel** [comment]
Nice job ! I think you can replace or add in doc-content-conversion-for-new-doc-site.adoc a paragraph to explain how and when run this script

**Comment by passga** [comment]
> Nice job ! I think you can replace or add in doc-content-conversion-for-new-doc-site.adoc a paragraph to explain how and when run this script

I have just amended the commit with the update of documentation

File: docs/migrate-from-old-documentation-site/doc-content-conversion-for-new-doc-site.adoc [hunk 1]
```diff
@@ -48,30 +48,34 @@ There is also a dependency on font-awesome css (`fa-` directives). Font awesome
 For such a page, the html part must be commented prior running conversion otherwise the process fails.
 ====
 
-Markdown source files are stored in the md folder of the repository. +
-From this folder, run the following command to generate the converted files:
+Thanks to the scripts link:../scripts/convert_md_to_asciidoc.py[convert_md_to_asciidoc.py], the conversion was done automatically.
```
Comment (alachambre) [inline docs/migrate-from-old-documentation-site/doc-content-conversion-for-new-doc-site.adoc:51]:
```suggestion
The script link:../scripts/convert_md_to_asciidoc.py[convert_md_to_asciidoc.py] performs the conversion automatically.
```

File: docs/migrate-from-old-documentation-site/doc-content-conversion-for-new-doc-site.adoc [hunk 2]
```diff
@@ -48,30 +48,34 @@ There is also a dependency on font-awesome css (`fa-` directives). Font awesome
...
+
+From the root old documentation  folder, run the following command
```
Comment (alachambre) [inline docs/migrate-from-old-documentation-site/doc-content-conversion-for-new-doc-site.adoc:53]:
```suggestion
From the root of the old documentation folder, run the following command
```

File: docs/migrate-from-old-documentation-site/doc-content-conversion-for-new-doc-site.adoc [hunk 3]
```diff
@@ -48,30 +48,34 @@ There is also a dependency on font-awesome css (`fa-` directives). Font awesome
...
-       bash -c 'kramdoc \
-               --format=GFM \
-               --output=@@.adoc @@';
+ python convert_md_to_asciidoc.py
 ----
 
-WARNING: test with additional `--wrap=ventilate` option
+.The script will :
+[info]
+* Sanitize the markdown source file to render it more easy to be convertible.
```
Comment (alachambre) [inline docs/migrate-from-old-documentation-site/doc-content-conversion-for-new-doc-site.adoc:61]:
```suggestion
* Sanitize the markdown source files to make theme convertible (update syntax). 
```

**Comment by tbouffard** [comment]
⚠️ may clash with the content of #82 - please merge #82 first

**Review by alachambre — APPROVED** [review]
works fine 👍

**Pull request merged**